ledger/complete/ledger.go

```diff
@@ -252,6 +253,10 @@ func (l *Ledger) ExportCheckpointAt(
 	// get trie
 	t, err := l.forest.GetTrie(ledger.RootHash(state))
 	if err != nil {
+		rh, _ := l.forest.MostRecentTouchedRootHash()
```

Comment:
I was thinking I would leave this in, so if you run state extraction with the wrong hash, you get the most recently used one.